diff --git a/dist/components/ConnectButton/ConnectButton.d.ts b/dist/components/ConnectButton/ConnectButton.d.ts
index 39a95f1cfcd2ae28058e52690320886cdcf0d94e..0a333dd50c2170ee336cddd69b33a151952b7013 100644
--- a/dist/components/ConnectButton/ConnectButton.d.ts
+++ b/dist/components/ConnectButton/ConnectButton.d.ts
@@ -1,6 +1,6 @@
 import React from 'react';
 import { ResponsiveValue } from '../../css/sprinkles.css';
-import { ConnectButtonRenderer } from './ConnectButtonRenderer';
+import { ConnectButtonRenderer } from './ConnectButtonRenderer.js';
 type AccountStatus = 'full' | 'avatar' | 'address';
 type ChainStatus = 'full' | 'icon' | 'name' | 'none';
 export interface ConnectButtonProps {
diff --git a/dist/components/index.d.ts b/dist/components/index.d.ts
index e6bf7a6270102d9e468f607699863a4081ffb7de..a6a3a19f24ba0b1bcc326b408a71e3308f22a878 100644
--- a/dist/components/index.d.ts
+++ b/dist/components/index.d.ts
@@ -1,2 +1,2 @@
-export { ConnectButton } from './ConnectButton/ConnectButton';
+export { ConnectButton } from './ConnectButton/ConnectButton.js';
 export { RainbowKitProvider } from './RainbowKitProvider/RainbowKitProvider';
diff --git a/dist/index.d.ts b/dist/index.d.ts
index 96a82b879c7249a5ac9b72128cd6a32792364ff6..168b9c68a79889a21bcb813cd436ea43eabe4fdf 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1,4 +1,6 @@
-export * from './components';
+export * from './components/index.js';
+export { ConnectButton } from './components/ConnectButton/ConnectButton.js'
+export { RainbowKitProvider } from './components/RainbowKitProvider/RainbowKitProvider.js'
 export { getDefaultWallets } from './wallets/getDefaultWallets';
 export { getWalletConnectConnector } from './wallets/getWalletConnectConnector';
 export { connectorsForWallets } from './wallets/connectorsForWallets';
