diff --git a/node_modules/@rainbow-me/rainbowkit/.DS_Store b/node_modules/@rainbow-me/rainbowkit/.DS_Store
new file mode 100644
index 0000000..d4c3117
Binary files /dev/null and b/node_modules/@rainbow-me/rainbowkit/.DS_Store differ
diff --git a/node_modules/@rainbow-me/rainbowkit/dist/index.d.ts b/node_modules/@rainbow-me/rainbowkit/dist/index.d.ts
index 96a82b8..b467d58 100644
--- a/node_modules/@rainbow-me/rainbowkit/dist/index.d.ts
+++ b/node_modules/@rainbow-me/rainbowkit/dist/index.d.ts
@@ -1,4 +1,6 @@
 export * from './components';
+export { ConnectButton } from './components/ConnectButton/ConnectButton.js'
+export { RainbowKitProvider } from './components/RainbowKitProvider/RainbowKitProvider.js'
 export { getDefaultWallets } from './wallets/getDefaultWallets';
 export { getWalletConnectConnector } from './wallets/getWalletConnectConnector';
 export { connectorsForWallets } from './wallets/connectorsForWallets';
