diff --git a/dist/esm/src/partials/w3m-connector-list/index.js b/dist/esm/src/partials/w3m-connector-list/index.js
index 2b0524b6fe9c4621f6dbf2ab58e8fb61c76e70b6..313ececdad2eef8a7e0fbaee61eafdc001b1c017 100644
--- a/dist/esm/src/partials/w3m-connector-list/index.js
+++ b/dist/esm/src/partials/w3m-connector-list/index.js
@@ -7,7 +7,7 @@ var __decorate = (this && this.__decorate) || function (decorators, target, key,
 import { customElement } from '@reown/appkit-ui';
 import { LitElement, html } from 'lit';
 import styles from './styles.js';
-import { ApiController, ChainController, ConnectorController, OptionsController, StorageUtil } from '@reown/appkit-core';
+import { ApiController, ChainController, ConnectorController, OptionsController, StorageUtil, CoreHelperUtil } from '@reown/appkit-core';
 import { ConstantsUtil as CommonConstantsUtil } from '@reown/appkit-common';
 import { state } from 'lit/decorators.js';
 import { WalletUtil } from '../../utils/WalletUtil.js';
@@ -26,8 +26,8 @@ let W3mConnectorList = class W3mConnectorList extends LitElement {
         const enableWalletConnect = OptionsController.state.enableWalletConnect;
         return html `
       <wui-flex flexDirection="column" gap="xs">
-        ${enableWalletConnect
-            ? html `<w3m-connect-walletconnect-widget></w3m-connect-walletconnect-widget>`
+        ${external.length
+            ? html `<w3m-connect-external-widget></w3m-connect-external-widget>`
             : null}
         ${recent.length ? html `<w3m-connect-recent-widget></w3m-connect-recent-widget>` : null}
         ${multiChain.length
@@ -42,10 +42,10 @@ let W3mConnectorList = class W3mConnectorList extends LitElement {
         ${featured.length
             ? html `<w3m-connect-featured-widget></w3m-connect-featured-widget>`
             : null}
-        ${custom?.length ? html `<w3m-connect-custom-widget></w3m-connect-custom-widget>` : null}
-        ${external.length
-            ? html `<w3m-connect-external-widget></w3m-connect-external-widget>`
+        ${enableWalletConnect
+            ? html `<w3m-connect-walletconnect-widget></w3m-connect-walletconnect-widget>`
             : null}
+        ${custom?.length ? html `<w3m-connect-custom-widget></w3m-connect-custom-widget>` : null}
         ${recommended.length
             ? html `<w3m-connect-recommended-widget></w3m-connect-recommended-widget>`
             : null}
@@ -56,11 +56,17 @@ let W3mConnectorList = class W3mConnectorList extends LitElement {
         const { featured, recommended } = ApiController.state;
         const { customWallets: custom } = OptionsController.state;
         const recent = StorageUtil.getRecentWallets();
+        const MetaMask = 'c57ca95b47569778a828d19178114f4db188b89b763c899ba0be274e97267d96';
+        const CoinBase = 'fd20dc426fb37566d803205b19bbc1d4096b248ac04548e3cfb6b3a38bd033aa';
+        const Rabby = '18388be9ac2d02726dbac9777c96efaac06d744b2f6d580fccdd4127a6d01fd1';
+        const OKX = '971e689d0a5be527bac79629b4ee9b925e82208e5168b733496a09c0faed0709'
+        const promotes = CoreHelperUtil.isMobile() ? [MetaMask, Rabby, OKX]: [CoinBase, Rabby, OKX]
+
         const filteredRecommended = WalletUtil.filterOutDuplicateWallets(recommended);
         const filteredFeatured = WalletUtil.filterOutDuplicateWallets(featured);
         const multiChain = this.connectors.filter(connector => connector.type === 'MULTI_CHAIN');
         const announced = this.connectors.filter(connector => connector.type === 'ANNOUNCED');
-        const injected = this.connectors.filter(connector => connector.type === 'INJECTED');
+        const injected = this.connectors.filter(connector => connector.type === 'INJECTED' && promotes.includes(connector.id));
         const external = this.connectors.filter(connector => connector.type === 'EXTERNAL');
         const isEVM = ChainController.state.activeChain === CommonConstantsUtil.CHAIN.EVM;
         const includeAnnouncedAndInjected = isEVM ? OptionsController.state.enableEIP6963 : true;
