diff --git a/dist/esm/src/partials/w3m-connect-injected-widget/index.js b/dist/esm/src/partials/w3m-connect-injected-widget/index.js
index 8834cda2346d8316a340b914f00e39bad9008192..0338aec6c21ea0395b85601a698807fcc8f0a16d 100644
--- a/dist/esm/src/partials/w3m-connect-injected-widget/index.js
+++ b/dist/esm/src/partials/w3m-connect-injected-widget/index.js
@@ -20,7 +20,12 @@ let W3mConnectInjectedWidget = class W3mConnectInjectedWidget extends LitElement
         this.unsubscribe.forEach(unsubscribe => unsubscribe());
     }
     render() {
-        const injectedConnectors = this.connectors.filter(connector => connector.type === 'INJECTED');
+        const MetaMask = 'io.metamask';
+        const CoinBase = 'coinbaseWalletSDK';
+        const Rabby = 'io.rabby';
+        const OKX = 'com.okex.wallet'
+        const RDNSs = CoreHelperUtil.isMobile() ? [CoinBase, OKX, MetaMask]: [MetaMask, Rabby, OKX]
+        const injectedConnectors = this.connectors.filter(connector => connector.type === 'INJECTED').filter(x => RDNSs.includes(x.id));
         if (!injectedConnectors?.length ||
             (injectedConnectors.length === 1 &&
                 injectedConnectors[0]?.name === 'Browser Wallet' &&
@@ -28,6 +33,7 @@ let W3mConnectInjectedWidget = class W3mConnectInjectedWidget extends LitElement
             this.style.cssText = `display: none`;
             return null;
         }
+
         return html `
       <wui-flex flexDirection="column" gap="xs">
         ${injectedConnectors.map(connector => {
diff --git a/dist/esm/src/partials/w3m-connect-walletconnect-widget/index.js b/dist/esm/src/partials/w3m-connect-walletconnect-widget/index.js
index a518866ac6bd84ec46379b4d1a8ab7794c628b02..2fd8e9e9aace23e872a278b78bd5fb4829daff9b 100644
--- a/dist/esm/src/partials/w3m-connect-walletconnect-widget/index.js
+++ b/dist/esm/src/partials/w3m-connect-walletconnect-widget/index.js
@@ -4,7 +4,7 @@ var __decorate = (this && this.__decorate) || function (decorators, target, key,
     else for (var i = decorators.length - 1; i >= 0; i--) if (d = decorators[i]) r = (c < 3 ? d(r) : c > 3 ? d(target, key, r) : d(target, key)) || r;
     return c > 3 && r && Object.defineProperty(target, key, r), r;
 };
-import { AssetUtil, ChainController, ConnectorController, CoreHelperUtil, RouterController } from '@reown/appkit-core';
+import { AssetUtil, ChainController, ConnectorController, RouterController } from '@reown/appkit-core';
 import { customElement } from '@reown/appkit-ui';
 import { LitElement, html } from 'lit';
 import { state } from 'lit/decorators.js';
@@ -20,10 +20,6 @@ let W3mConnectWalletConnectWidget = class W3mConnectWalletConnectWidget extends
         this.unsubscribe.forEach(unsubscribe => unsubscribe());
     }
     render() {
-        if (CoreHelperUtil.isMobile()) {
-            this.style.cssText = `display: none`;
-            return null;
-        }
         const connector = this.connectors.find(c => c.id === 'walletConnect');
         if (!connector) {
             this.style.cssText = `display: none`;
diff --git a/dist/esm/src/partials/w3m-connector-list/index.js b/dist/esm/src/partials/w3m-connector-list/index.js
index 2b0524b6fe9c4621f6dbf2ab58e8fb61c76e70b6..002edc8fab5048a5396ff19ffd892708a974a0e4 100644
--- a/dist/esm/src/partials/w3m-connector-list/index.js
+++ b/dist/esm/src/partials/w3m-connector-list/index.js
@@ -26,8 +26,8 @@ let W3mConnectorList = class W3mConnectorList extends LitElement {
         const enableWalletConnect = OptionsController.state.enableWalletConnect;
         return html `
       <wui-flex flexDirection="column" gap="xs">
-        ${enableWalletConnect
-            ? html `<w3m-connect-walletconnect-widget></w3m-connect-walletconnect-widget>`
+        ${external.length
+            ? html `<w3m-connect-external-widget></w3m-connect-external-widget>`
             : null}
         ${recent.length ? html `<w3m-connect-recent-widget></w3m-connect-recent-widget>` : null}
         ${multiChain.length
@@ -42,10 +42,10 @@ let W3mConnectorList = class W3mConnectorList extends LitElement {
         ${featured.length
             ? html `<w3m-connect-featured-widget></w3m-connect-featured-widget>`
             : null}
-        ${custom?.length ? html `<w3m-connect-custom-widget></w3m-connect-custom-widget>` : null}
-        ${external.length
-            ? html `<w3m-connect-external-widget></w3m-connect-external-widget>`
+        ${enableWalletConnect
+            ? html `<w3m-connect-walletconnect-widget></w3m-connect-walletconnect-widget>`
             : null}
+        ${custom?.length ? html `<w3m-connect-custom-widget></w3m-connect-custom-widget>` : null}
         ${recommended.length
             ? html `<w3m-connect-recommended-widget></w3m-connect-recommended-widget>`
             : null}
@@ -56,6 +56,7 @@ let W3mConnectorList = class W3mConnectorList extends LitElement {
         const { featured, recommended } = ApiController.state;
         const { customWallets: custom } = OptionsController.state;
         const recent = StorageUtil.getRecentWallets();
+
         const filteredRecommended = WalletUtil.filterOutDuplicateWallets(recommended);
         const filteredFeatured = WalletUtil.filterOutDuplicateWallets(featured);
         const multiChain = this.connectors.filter(connector => connector.type === 'MULTI_CHAIN');
